wrappers/nodejs/src/addon.cpp

```diff
@@ -4696,6 +4696,7 @@ void InitModule(v8::Local<v8::Object> exports) {
   _FORCE_SET_ENUM(RS2_OPTION_NOISE_FILTERING);
   _FORCE_SET_ENUM(RS2_OPTION_INVALIDATION_BYPASS);
   _FORCE_SET_ENUM(RS2_OPTION_AMBIENT_LIGHT);
+  _FORCE_SET_ENUM(RS2_OPTION_DIGITAL_GAIN);
```

Comment:
This works, with the same value as the prev line?

Reply:
I haven't tested it but I looked at the logic and it seems OK.
I define a pair of name to value map, the name is the key

#define _FORCE_SET_ENUM(name) \
  Nan::DefineOwnProperty(exports, \
      Nan::New(#name).ToLocalChecked(), \
      Nan::New(static_cast<int>((name))), \
      static_cast<v8::PropertyAttribute>(v8::ReadOnly | v8::DontDelete));